rosdep/base.yaml

```diff
@@ -1580,6 +1580,11 @@ gradle:
   nixos: [gradle]
   ubuntu: [gradle]
 graphicsmagick:
+  alpine:
+    '*': [graphicsmagick, graphicsmagick-dev]
+    '3.11': null
+    '3.14': null
```

Comment:
Seems available on 3.11 and 14
https://pkgs.alpinelinux.org/package/v3.11/community/x86_64/graphicsmagick
https://pkgs.alpinelinux.org/package/v3.14/community/x86_64/graphicsmagick